react-js/react.md

```diff
@@ -145,7 +80,7 @@ will render as:
 ### Independent Practice
 - [CodePen for Hello World Example](https://codepen.io/jamesjose03/pen/gOpVeLQ)
 - Other Requirements: 
-  - Modify the state or components.
+  - Modify the props or components.
   - Play along with the code, modify it and see the changes it makes.
 
```

Comment:
This is not independent practice really. Why would they go back again to HelloWorld? 

This was the issue description for independent practice...
>  Use one of the CodePen links in "Lesson Materials" and add some required and optional requirements for them to implement

What is the goal of this practice? Give them an end goal but do not specify how to accomplish it. In my comments that you resolved, I said that extracting the map callback as a separate component would be a good example but you're free to use another goal of your choosing. Ideally, give them a choice of a few.

Reply:
👍